src/Microsoft.AspNetCore.OData/ODataController.cs

```diff
@@ -51,5 +51,62 @@ protected virtual UpdatedODataResult<TEntity> Updated<TEntity>(TEntity entity)
 
             return new UpdatedODataResult<TEntity>(entity);
         }
+
+        /// <summary>
+        /// Creates a <see cref="StatusCodeResult"/> that when executed will produce a Bad Request (400) response.
+        /// </summary>
+        /// <returns>A <see cref="BadRequestResult"/> with the specified values.</returns>
+        public override BadRequestResult BadRequest()
```

Comment:
>BadRequest() [](start = 41, length = 12)

So, Could we do like this:

public override ODataErrorResult ODataErrorRequest(string message, int stateCode)

Reply:
#Resolved